src/ui/public/directives/config.js

```diff
@@ -8,75 +7,110 @@ var module = uiModules.get('kibana');
 
 
 /**
- * config directive
+ * kbnTopNav directive
  *
- * Creates a full width horizonal config section, usually under a nav/subnav.
+ * The top section that shows the timepicker, load, share and save dialogues.
  * ```
- * <config config-template="configTemplate" config-object="configurable"></config>
+ * <kbn-top-nav name="current-app-for-extensions" config="path.to.menuItems"></kbn-top-nav>
  * ```
  */
 
-module.directive('config', function ($compile) {
+module.directive('kbnTopNav', function (Private) {
```

Comment:
It does have tests. Not the most comprehensive, but they exist. I just check for the existence of the functions is, open, close, and toggle. Should I add more?